### PR TITLE
Update the GSP introductory notes for UK and XI.

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
[HOTT-1218](https://transformuk.atlassian.net/browse/HOTT-1218)

### What?
Update the GSP introductory notes for UK and XI.

### Before
Screenshot 2022-02-22 at 10 21 15

### After
Screenshot 2022-02-22 at 10 45 10

### CODE REVIEW

File: db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md [hunk 1]
```diff
@@ -1 +1,159 @@
...
+(a) through working or processing a maximum content of non-originating materials is not exceeded;
+
+(b) through working or processing the 4-digit Harmonized System heading or 6-digit Harmonized System sub-heading of the manufactured products becomes different from the 4-digit Harmonized System heading or 6-digit sub-heading respectively of the materials used;
+
+(c) a specific working and processing operation is carried out;
+
+(d) working or processing is carried out on certain wholly obtained materials.
+
+### Note 2: The structure of the list
+2.1. Columns 1 and 2 describe the product obtained. Column 1 gives the chapter number, 4-digit heading or 6-digit sub-heading number used in the Harmonized System, as appropriate. Column 2 gives the description of goods used in that system for that heading or chapter. For each entry in columns 1 and 2, subject to Note 2.4, one or more rules (“qualifying operations”) are set out in column 3. These qualifying operations concern only non-originating materials. Where, in some cases, the entry in column 1 is preceded by “ex”, this signifies that the rule in column 3 applies only to the part of that heading as described in column 2.
```
Comment (jebw) [inline db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md:16]: Think there's a new line needed here

File: db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md [hunk 2]
```diff
@@ -1 +1,159 @@
...
+
+7.2. Without prejudice to Note 7.3, materials, which are not classified within Chapters 50 to 63, may be used freely in the manufacture of textile products, whether or not they contain textiles.
+
+Example:
+If a rule in the list provides that, for a particular textile item (such as trousers), yarn must be used, this does not prevent the use of metal items, such as buttons, because buttons are not classified within Chapters 50 to 63. For the same reason, it does not prevent the use of slide-fasteners, even though slide-fasteners normally contain textiles.
+
+7.3. Where a percentage-rule applies, the value of materials which are not classified within Chapters 50 to 63 must be taken into account when calculating the value of the non-originating materials incorporated.
+
+### Note 8 —Definition of specific processes and simple operations carried out in respect of certain products of Chapter 27
+8.1. For the purposes of headings ex 2707 and 2713, the “specific processes” are the following:
```
Comment (jebw) [inline db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md:129]: likewise a new line needed here

**Review by jebw — CHANGES_REQUESTED** [review]
Looks good - just needs the missing new lines adding in :+1:

**Comment by alexdesi** [comment]
It seems there is a bug in there, I replaced both files in the relevant folders, but the commodity pages show the same Introductory notes for both XI and UK (which is the gps_intro_uk)

I am fixing it.

File: db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md (hunk 1 above)
Comment (alexdesi) [reply db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md:17]: 👍 fixed, probably it renders the same, but it neater with the new line.

File: db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md (hunk 2 above)
Comment (alexdesi) [reply db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md:132]: done

**Pull request merged**